blockchain/blockchain.go

```diff
@@ -8,7 +8,7 @@
 //
 // When used as a standalone application, this package implements the Flow Access API
 // and is fully-compatible with Flow gRPC client libraries.
-package emulator
+package blockchain
```

Comment:
Not coming from your changes, but could you move up the `NewBlockchain` method so it's not buried in between